dm/worker/worker.go

```diff
@@ -103,6 +106,9 @@ func (w *Worker) Start() {
 	// start purger
 	w.relayPurger.Start()
 
+	// start tracer
+	w.tracer.Start()
```

Comment:
1. seems not to call `Stop`?
2. always `Start` tracing even `Tracer.Enable` is `false`, and let lazy judge until try to send trace event? (later we can switch `Enable` online?)

Reply:
There will be online switch `Enable` for tracing by design, but now just check `Enable` before a tracer `Start`. I will refine it if needed